protocols/kad/src/behaviour.rs

```diff
@@ -1455,37 +1437,43 @@ where
                     }
                 }
 
-                Some(KademliaEvent::OutboundQueryCompleted {
+                Some(KademliaEvent::OutboundQueryProgressed {
                     id: query_id,
                     stats: result.stats,
                     result: QueryResult::Bootstrap(Err(BootstrapError::Timeout {
                         peer,
                         num_remaining,
                     })),
+                    step: ProgressStep::single(),
```

Comment:
There might have been `OutboundQueryProgressed` events emitted for this `bootstrap` call.

Reply:
Would be fixed with https://github.com/libp2p/rust-libp2p/commit/cc1b2bdfc09a89a56318e38861204ed2c204cfb8.

Reply:
should be fixed